shared_model/validators/validators_common.hpp

```diff
@@ -17,13 +17,19 @@ namespace shared_model {
      */
     struct ValidatorsConfig {
       ValidatorsConfig(uint64_t max_batch_size,
-                       bool partial_ordered_batches_are_valid = false);
+                       bool partial_ordered_batches_are_valid = false,
+                       bool duplicate_tolerance = true);
```

Comment:
Why did you decide to put `true` as a default value?

Duplicates are allowed only within a proposal. All the rest cases deny transactions duplication.

Please change the default value to false and correct related code in other files if needed.

Reply:
Fixed, and added new config for proposal validation